e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -492,25 +492,17 @@ describeEE("scenarios > embedding > full app", () => {
           },
         });
       });
-      cy.get("@postMessage")
-        .should("have.been.calledWith", {
-          metabase: {
-            type: "frame",
-            frame: {
-              mode: "fit",
-              height: Cypress.sinon.match(value => value > 1000),
-            },
-          },
-        })
-        .and("not.have.been.calledWith", {
-          metabase: {
-            type: "frame",
-            frame: {
-              mode: "fit",
-              height: Cypress.sinon.match(value => value < 400),
-            },
+
+      // TODO: Find a way to assert that this is the last call.
```

Comment:
The previous assertion was trying to ensure that the dashboard postmessage won't be called with a shorter height. However, since May 29, this doesn't seem to be the case. It seems in the flake the dashboard only renders with their header, and this postmessage has been called with the height around ~150px. That doesn't totally break the feature though because at the end the postmessage will be called again with the correct height.

I wish there's a way to assert that eventually we called the post message with the height greater than 1000px (for the first tab)